homeassistant/components/device_tracker/__init__.py

```diff
@@ -6,7 +6,12 @@
 from homeassistant.helpers.typing import ConfigType
 from homeassistant.loader import bind_hass
 
-from .config_entry import async_setup_entry, async_unload_entry  # noqa: F401
+from .config_entry import (  # noqa: F401
+    BaseTrackerEntity,
```

Comment:
We should not expose the BaseTrackerEntity.

Reply:
done